lib/matplotlib/backend_bases.py

```diff
@@ -3606,18 +3604,41 @@ def _tool_triggered_cbk(self, event):
 
         self.toggle_toolitem(event.tool.name)
 
-    def _add_tool_cbk(self, event):
-        """Captures 'tool_added_event' and adds the tool to the toolbar"""
-        image = self._get_image_filename(event.tool.image)
-        toggle = getattr(event.tool, 'toggled', None) is not None
-        self.add_toolitem(event.tool.name,
-                          event.data['group'],
-                          event.data['position'],
-                          image,
-                          event.tool.description,
-                          toggle)
+    def add_tools(self, tools):
+        """ Add multiple tools to `Navigation`
+
+        Parameters
+        ----------
+        tools : List
+            List in the form
+            [[group1, [name1, name2 ...]][group2...]]
+            where group1 is the name of the group where the
+            Tool1, Tool2... are going to be added, and name1, name2... are the
+            names of the tools
+        """
+
+        for group, grouptools in tools:
+            for position, tool in enumerate(grouptools):
+                self.add_tool(tool, group, position)
+
+    def add_tool(self, tool, group, position):
```

Comment:
Because we pass the name to ``Navigation.get_tool``, so we don't want to give that as the kwarg...

Sometimes, as I did here, I like to use tool, as an abstract word, it could mean name, or it could mean the object, or the class.  So now we say no object, we definitely need a `name`, but we don't necessarily need a `tool_class`